apps/recipes-react-components/src/recipes/media-object/MediaObject.stories.mdx

```diff
@@ -0,0 +1,222 @@
+<Meta title="Concepts/Recipes/Media Object" />
+
+---
+
+# Media Object recipe
+
+### **Overview**
+
+A Media Object, is a pattern that is commonly used to display some media content and a description of that content.
+It is commonly used to display an Avatar/Image of a person along with their name and description. We provide an
+out of the box component in Fluent UI (v8's Persona and v9's Persona) that can be used to achieve this pattern.
+
+This recipe focuses on the other uses of this pattern, where media content is not a persona. We will guide you on
+how to create a Media Object that can be used to display images, icons, and any other media content you might
+want to add.
+
+### **Ingredients**
+
+- <LinkTo kind="concepts-developer-icons-icons--page">@fluentui/react-icons</LinkTo>
+- <LinkTo kind="components-image--default">@fluentui/react-image</LinkTo>
+- <LinkTo kind="components-text--default">@fluentui/react-text</LinkTo>
+
+> **\*\*Note:\*\*** This recipe is meant to provide a way to use custom media. If you want to use
+> `Avatar` or `PresenceBadge` as media, please refer to our <LinkTo kind="components-persona--default">Persona component</LinkTo>.
+
+## **Steps**
+
+There are two ways to build a Media Object, you can use a grid layout or a flex layout. We will cover both in this
+recipe. There are upsides and downsides to both, but the main difference is that grid layout uses less dom than the
+flex option.
+
+### **Flex Layout**
+
+The main idea behind the flex layout is to have a container and two children, as shown below. The parent div will
+contain the media and another div (text div) that will contain the text content.
+
+<Example centered>
+  <FlexSkeleton />
+</Example>
+
+To achieve this layout, we will follow these steps:
+
+#### **Step 1: Creating our parent and text container styles.**
+
+As seen above, we will first need to create our parent div and our text containers. These containers will need to have
+the `display: flex` css property and a `row` direction for the parent and a `column` direction for the text container, as shown in the `makeStyles` call below:
+
+```jsx
+const useStyles = makeStyles({
+  parent: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  text: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+});
+```
+
+#### **Step 2: Putting everything together.**
+
+After we have our styles, we just have to put our styles to work! Since we are using flex box we don't need to worry about dynamically placing items. Do take into account that using this approach will be heavier on the dom than the grid approach.
+
+```jsx
+const MediaObject: React.FC<{}> = ({ children }) => {
+  const styles = useStyles();
+
+  return (
+    <div className={styles.parent}>
+      {/** Media goes here 🖼️ **/}
+      <div className={styles.text}>{/** Text lines go here 📝 **/}</div>
+    </div>
+  );
+};
+```
+
+After these steps you should be able to build something like the example below.
+
+<Example centered>
```

Comment:
This is to make the rendered example centered.

![image](https://user-images.githubusercontent.com/5953191/217045747-3bb05e09-071f-445e-a276-5c306348478b.png)

I can remove it thought it would be nice to add for future recipes since if it's not there, then we need to add another div to center the example.

Reply:
Ah, I see. It's not in the example guide but it is in the actual example code. My thinking is that I'd expect to be able to copy-paste any of the examples and have them work. I find it a little confusing that `centered` shows up in the example but the styles/props are not used anywhere in the guide.